modules/brainbrowser/templates/form_brainbrowser.tpl

```diff
@@ -1,135 +1,209 @@
+                        <div class="control-heading" id="slice-series-heading-{{VOLID}}">View Slices</div>
+                        <div class="slice-series-buttons">
+                            <span class="slice-series-button button btn btn-sm btn-primary"
+                                data-axis="xspace">Sagittal</span>
+                            <span class="slice-series-button button btn btn-sm btn-primary"
+                                data-axis="yspace">Coronal</span>
+                            <span class="slice-series-button button btn btn-sm btn-primary"
+                                data-axis="zspace">Transverse</span>
+                        </div>
+                    </div>
+                </div>
+                <!--closing filename-additional-info -->
             </div>
-        <div class="line-separator"></div>
-        <div id="intensity-value-div-{{VOLID}}">
-            <span class="control-heading" data-volume-id="{{VOLID}}">
-               Intensity Value
-            </span>
-            <span id="intensity-value-{{VOLID}}" class="intensity-value control-inputs"></span>
-            <span id="intensity-value-bg-{{VOLID}}" class="intensity-value control-inputs" style="height:20px"></span>
         </div>
-        <div class="line-separator"></div>
-        <div id="time-{{VOLID}}" class="time-div" data-volume-id="{{VOLID}}" style="display: none">
-            <span class="control-heading">Time</span>
-            <input class="control-inputs time-inputs" value="0" id="time-val-{{VOLID}}"/>
-            <span class="btn btn-sm btn-primary play-btn">
-            <input type="checkbox" class="button ui-helper-hidden-accessible" id="play-{{VOLID}}"><label for="play-{{VOLID}}">Play</label>
-            </span>
-            <div class="slider volume-viewer-threshold" id="threshold-time-slider-{{VOLID}}"></div>
-            <div class="line-separator"></div>
         </div>
-        <div id="slice-series-{{VOLID}}" class="slice-series-div" data-volume-id="{{VOLID}}">
-            <div class="control-heading" id="slice-series-heading-{{VOLID}}">View slices</div>
-            <span class="slice-series-button button btn btn-sm btn-primary" data-axis="xspace">Sagittal</span>
-            <span class="slice-series-button button btn btn-sm btn-primary" data-axis="yspace">Coronal</span>
-            <span class="slice-series-button button btn btn-sm btn-primary" data-axis="zspace">Transverse</span>
+
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+<div class="form-group col-sm-0 col-md-4">&nbsp;</div>
+
```

Comment:
@jstirling91 I don't know how else to stop the images from being on the same row as the menu
I tried to use div class="row", but it's not working..